interp/notation.ml

```diff
@@ -659,6 +662,10 @@ let rec glob_of_constr token_kind ?loc env sigma c = match Constr.kind c with
   | Ind (ind, _) -> DAst.make ?loc (Glob_term.GRef (IndRef ind, None))
   | Var id -> DAst.make ?loc (Glob_term.GRef (VarRef id, None))
   | Int i -> DAst.make ?loc (Glob_term.GInt i)
+  | Sort Sorts.SProp -> DAst.make ?loc (Glob_term.GSort Glob_term.GSProp)
+  | Sort Sorts.Prop -> DAst.make ?loc (Glob_term.GSort Glob_term.GProp)
+  | Sort Sorts.Set -> DAst.make ?loc (Glob_term.GSort Glob_term.GSet)
+  | Sort (Sorts.Type _) -> DAst.make ?loc (Glob_term.GSort (Glob_term.GType [])) (* FIXME: explain why this list can be empty if its not obvious *)
```

Comment:
It seems dubious to add code with a FIXME in it. Shall it be addressed before this PR be merged?

Reply:
@vbgl Sorry for the very late reply.  I left this because it's not clear to me why this list can be empty.  If one of the other devs has an explanation (or claims it's obvious), then I will remove the FIXME.  If no-one can explain it, but it works, then it seems reasonable the leave the FIXME?

Reply:
It seems reasonable to me. Maybe Gaëtan can provide an explanation (@SkySkimmer).

Reply:
This doesn't work anymore as we changed the glob sort representation in https://github.com/coq/coq/pull/8988